string drone_id
float64 height
float64 velocity
